packages/cozy-client/src/models/paper.js

```diff
@@ -134,17 +139,13 @@ export const computeExpirationNoticeDate = file => {
 /**
  * @param {IOCozyFile} file - io.cozy.files document
  * @returns {string | null} Expiration notice link
- * @description Computes et returns the expiration notice link of the given file, or null if it has none or it is not expiring
+ * @description Computes et returns the expiration notice link of the given file, or null if it has none
  */
 export const computeExpirationNoticeLink = file => {
   const qualificationLabel = file.metadata?.qualification?.label
-  if (isExpiringFrenchNationalIdCard(file)) {
-    return 'https://www.service-public.fr/particuliers/vosdroits/N358'
-  }
-  if (isExpiringGeneric(file) && qualificationLabel === 'residence_permit') {
-    return 'https://www.service-public.fr/particuliers/vosdroits/N110'
-  }
-  return null
+  if (!qualificationLabel) return null
+
+  return EXPIRATION_LINK_BY_LABEL[qualificationLabel] ?? null
```

Comment:
nit: Is there any value here to use `??` instead of `||`? I don't encounter the former very often and I find it adds unecessary extra brain processing ;)

Reply:
Ici, l'un ou l'autre donnera le même résultat en effet